push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/PushyApnsSender.java

```diff
@@ -44,10 +46,11 @@
 import static org.jboss.aerogear.unifiedpush.message.util.ConfigurationUtils.tryGetIntegerProperty;
 import static org.jboss.aerogear.unifiedpush.message.util.ConfigurationUtils.tryGetProperty;
 
+@Stateless
 @SenderType(VariantType.IOS)
 public class PushyApnsSender implements PushNotificationSender {
 
-    private final Logger logger = LoggerFactory.getLogger(PushyApnsSender.class);
+    private static final Logger logger = LoggerFactory.getLogger(PushyApnsSender.class);
```

Comment:
Static reference to slf4j should be avoided: https://wiki.apache.org/commons/Logging/StaticLog